packages/builder/src/pages/builder/workspace/[application]/agent/index.svelte

```diff
@@ -142,7 +144,7 @@
       await API.agentChatStream(
         updatedChat,
         $params.application,
-        chunk => {
+        (chunk: any) => {
```

Comment:
This will be removed shortly as part of the ai work so didn't to spend any time trying to type out, as it's quite complicated.